pkg/edit/vars_test.go

```diff
@@ -47,3 +67,25 @@ func TestAddVars(t *testing.T) {
 		That("add-vars [&a~='']").Throws(ErrorWithType(vals.WrongType{})),
 	)
 }
+
+func TestDelVars(t *testing.T) {
+	TestWithSetup(t, func(ev *eval.Evaler) {
+		ev.ExtendGlobal(eval.BuildNs().AddGoFn("add-vars", addVars))
+		ev.ExtendGlobal(eval.BuildNs().AddGoFn("del-vars", delVars))
+	},
+		That("add-vars [&foo=bar]").Then("del-vars [foo]").Then("put $foo").DoesNotCompile("variable $foo not found"),
+		That("add-vars [&a=A &b=B &c=C]").Then("del-vars [a b]").Then("put $c").Puts("C").Then("del-vars [c]").Then("put $c").DoesNotCompile("variable $c not found"),
```

Comment:
This test case seems to be testing two behaviors:

1. That deleting variables `a` and `b` leaves the variable `c` intact;
2. That deleting variable `c` makes it no longer exist.

Behavior 2 is already tested above and doesn't need to be repeated.